cmd/meroxa/builder/builder.go

```diff
@@ -666,10 +666,6 @@ func buildCommandWithFeatureFlag(cmd *cobra.Command, c Command) {
 		return
 	}
 
-	// Considering a command with feature flags not ready to be documented and
-	// publicly available.
-	cmd.Hidden = true
```

Comment:
I believe this should cause on building a new binary (running `make`) to generate documentation for all environments commands. They should all be added to this PR, right?

Reply:
Yes, I missed that. Adding.